controller/handlers.go

```diff
@@ -0,0 +1,39 @@
+package controller
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"wizeline/usecase"
+)
+
+func UserHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	users , err := usecase.GetAllUsers()
+	if err == nil {
```

Comment:
Instead of having if-else sentence, you could do something like this:

    if err != nil {
            w.WriteHeader(http.StatusInternalServerError)
		log.Println(err)
		http.Error(w,"Error reading users csv file",http.StatusInternalServerError)
		return
	}
    _ ,err := fmt.Fprintf(w, users)
	if err != nil{
		w.WriteHeader(http.StatusInternalServerError)
		log.Println(err)
		http.Error(w,"Error getting users",http.StatusInternalServerError)
		return
	}
	w.WriteHeader(http.StatusOK)